mods/uxtheme-hook.wh.cpp

```diff
@@ -96,7 +104,7 @@ VOID WINAPI Element_PaintBgHook(class Element* This, HDC hdc, class Value* value
 using SetSysColors_t = decltype(&SetSysColors);
 SetSysColors_t SetSysColors_orig;
 
-int SetSysColors_hook(int cElements, const INT *lpaElements, const COLORREF *lpaRgbValues)
+int STDCALL SetSysColors_hook(int cElements, const INT *lpaElements, const COLORREF *lpaRgbValues)

```

Comment:
It should be WINAPI. BTW it would be best if you could test it on 32-bit and make sure that it works and doesn't crash.

Reply:
but most of the mods, currently on windhawk, use this convention

Reply:
Yes, `WINAPI` is defined as `__stdcall` so they're the same, but it's more correct to use `WINAPI` for WinAPI functions such as `SetSysColors`. Which mod uses something other than `WINAPI` for WinAPI hooks?

Reply:
Oh okay, ill change it to WINAPI.